test/controllers/api_users/permissions_controller_test.rb

```diff
@@ -26,6 +26,69 @@ class ApiUsers::PermissionsControllerTest < ActionController::TestCase
       assert_not_authorised
     end
 
+    should "raise an exception if the user doesn't have access to the application" do
+      sign_in create(:admin_user)
```

Comment:
Only superadmins can access these routes, so we'll need to use a superadmin here